server/models/parentModel.js

```diff
@@ -1,16 +1,46 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
+//Joi is considered one of the most powerul schema desceiption language and DATA vlidator for JS.
+//Read more here : https://www.npmjs.com/package/joi 
 const Joi = require('joi');
+//passwordComplexity creates a joi object  https://www.npmjs.com/package/joi-password-complexity where 
+//we can set different requirement fields for our username, password. 
+const passwordComplexity = require("joi-password-complexity");
+//JSON webtoken - Read more about here https://www.npmjs.com/package/jsonwebtoken  (Remember, not recommended to use alone as a validator)
+const jwt = require('jsonwebtoken');
 
 const parentSchema = new Schema ({
   firstName: {type: String, required: true},
   lastName: {type: String, required: true},
-  email: {type: String, required: true},
+  email: {type: String, required: true},     //How can we implement username? 
```

Comment:
Great question!  Maybe we can add that as a stretch feature.  For the MVP, the parent will login with a username as password.  If they don't have an account set up, they sign up with an email, student ID, and password.  Please refer to the attached screenshots for an example of the login and signup pages.
<img width="776" alt="image" src="https://user-images.githubusercontent.com/105131644/188314564-bda1b293-2244-4d0a-be1b-52a759406ec5.png">
<img width="1059" alt="image" src="https://user-images.githubusercontent.com/105131644/188314583-90653199-5e97-4469-b6f5-247da3bc2f56.png">